Fibonacci_numbers_par.rb

```diff
@@ -0,0 +1,10 @@
+a = 0
```

Comment:
what is a, b and c? 

again a method should be name after what it does, a var should be name after what it means. 

lets review how we can achieve this result handling another ruby methods like `injection` or maybe something new with latest ruby version